jwk/x_manager_sql_migrations_test.go

```diff
@@ -58,7 +57,7 @@ func TestXXMigrations(t *testing.T) {
 		migratest.MigrationSchemas{createMigrations},
 		x.CleanSQL,
 		x.CleanSQL,
-		func(t *testing.T, db *sqlx.DB, k, m, steps int) {
+		func(t *testing.T, driver string, db *sqlx.DB, k, m, steps int) {
```

Comment:
Maybe rename this to `dbName`, otherwise this would be considered the driver which would be expected to be `postgres` for CockroachDB.